project3/styles.css

```diff
@@ -0,0 +1,321 @@
+*,
+*::before,
+*::after {
+    box-sizing: border-box;
+}
+
+body {
+    background-color: #163346;
+}
+
+form {
```

Comment:
Great job styling the `form` as a container/wrapper element without adding extra markup 👍